test/e2e/framework/framework.go

```diff
@@ -55,6 +56,8 @@ type Framework struct {
 	Clientset_1_2 *release_1_2.Clientset
 	Clientset_1_3 *release_1_3.Clientset
 
+	FederationClientset *federation_internalclientset.Clientset
+	// TODO: remove FederationClient, all the client access must be through FederationClientset
```

Comment:
@madhusudancs there are only two e2e tests now, simply switching from the client to the clientset would be a handful of additional lines in the diff. imho it's easier to just do this now rather than letting the deprecated field hang around and possibly attract more dependencies or subtle interactions as federation e2e grows.